code/design_pattern/strategy_pattern/strategy_pattern.cpp

```diff
@@ -0,0 +1,141 @@
+#include <iostream>
+using namespace std;
+
+class IFlyBehavior
+{
+	public :
```

Comment:
The word public should be at the same level as the class, as per the coding style guidelines.